app/experimenter/experiments/api/v5/serializers.py

```diff
@@ -348,8 +348,8 @@ def __init__(self, instance=None, data=None, **kwargs):
         super().__init__(instance=instance, data=data, **kwargs)
 
     def validate_publish_status(self, publish_status):
-        if (
-            publish_status == NimbusExperiment.PublishStatus.APPROVED
+        if publish_status == NimbusExperiment.PublishStatus.APPROVED and (
+            self.instance.publish_status != NimbusExperiment.PublishStatus.IDLE
```

Comment:
Yeah we just gotta remember to take this out in https://github.com/mozilla/experimenter/issues/4800